CHANGELOG.md

```diff
@@ -113,6 +113,7 @@
 - Added `Phalcon\Text::underscore` - to make a phrase underscored instead of spaced
 - Added `Phalcon\Text::humanize` - to make an underscored or dashed phrase human-readable
 - Added ability to change document class to be returned in ODM through `class` option
+- Added new getter `Phalcon\Mvc\Model\Query\Builder::getJoins()` - to get join parts from query builder
```

Comment:
Why you added it in 2.0.9 version ? :D I meant add it as last item in 2.1.x version, you previously added it in beginning of 2.1.x version @sergeyklay